### ISSUE TITLE
Buggy AXValueCreate, AXValueGetValue

### ISSUE THREAD

**Issue by quirm** [ronaldoussoren/pyobjc#361]
Hi, I'm trying to retrieve values or create AXValueRef to pass it further to other API functions. I did not find a way or docs how to correctly use `AXValueCreate` or `AXValueGetValue`.

I have python 3.8 and pyobjc 7.2.

When I checked [test_axvalue.py#L32-L37](https://github.com/ronaldoussoren/pyobjc/blob/master/pyobjc-framework-ApplicationServices/PyObjCTest/test_axvalue.py#L32-L37), it's also failing:

```
Traceback (most recent call last):
  File "axvalue.py", line 25, in <module>
    val = HIServices.AXValueCreate(HIServices.kAXValueTypeCGPoint, (1, 2))
ValueError: depythonifying 'unsigned long', got 'tuple'
```

For `AXValueGetValue`, it fails with a segmentation fault:

```
err, value_ref = HIServices.AXUIElementCopyAttributeValue(
    win_ref, 'AXPosition', None)
print(value)

err, res = HIServices.AXValueGetValue(value_ref, HIServices.kAXValueTypeCGPoint, None)
print(err, res)
```

Looking at [_HIService.m](https://github.com/ronaldoussoren/pyobjc/blob/master/pyobjc-framework-ApplicationServices/Modules/_HIServices.m#L9) it looks like there is implementation to handle it. Is it a bug or am I using it wrong?

**Comment by ronaldoussoren** [comment]
The _HIService extension was introduced after PyObjC 7.2.  I want to do a release that has this change in it soonish, but need to run the full testsuite on a number of systems before I do that.

**Comment by ronaldoussoren** [comment]
For background: most functions in the framework bindings are wrapped using dynamically created bindings. The machinery for handling these dynamically created functions is too limited to handle AXValueCreate and AXValueGetValue, hence the manual bindings in the upcoming release.

**Comment by quirm** [comment]
Oh, I have not realized it is only 9 days old, I see now #260. Thanks for the info.

**Comment by ronaldoussoren** [comment]
This was fixed in the 7.3 release.
